src/olympia/accounts/views.py

```diff
@@ -220,29 +203,9 @@ def inner(self, request):
                     request, ERROR_NO_PROFILE, next_path=next_path,
                     format=format)
             else:
-                identity_user = find_user(identity)
-                if request.user.is_authenticated():
-                    if (identity_user is not None and
-                            identity_user != request.user):
-                        log.info('Conflict finding user during FxA login. '
-                                 'request.user: {}, identity_user: {}'.format(
-                                     request.user.pk, identity_user.pk))
-                        return render_error(
-                            request, ERROR_USER_MISMATCH, next_path=next_path,
```

Comment:
Since the migration is over we no longer allow users to log in while already logged in. So the condition for this `request.user.is_authenticated()` has moved up and now renders a "you are already logged in" error. So I think this can go away.

I should remove the `ERROR_USER_MISMATCH` related stuff though.